pkg/common/funcs.go

```diff
@@ -18,16 +18,10 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-func InitContainerArgs(gke bool) []string {
-	if gke {
-		return []string{
-			"-c",
-			fmt.Sprintf("echo \"Running %[1]s\"; %[1]s", FalconDaemonsetInitBinary),
-		}
-	}
+func InitContainerArgs() []string {
 	return []string{
 		"-c",
-		fmt.Sprintf("echo \"Running %[1]s\"; %[1]s; test -f \"%[2]s\" && %[2]s || echo \"%[2]s not found. Skipping.\"", FalconDaemonsetInitBinary, FalconDaemonsetConfigureClusterIdBinary),
+		fmt.Sprintf("echo \"Running %[1]s\"; %[1]s; echo \"Running %[2]s\"; test -f \"%[2]s\" && %[2]s || echo \"%[2]s not found. Skipping.\"", FalconDaemonsetInitBinary, FalconDaemonsetConfigureClusterIdBinary),
```

Comment:
question: This seems a little unrelated to the GKE allow list changes. Were we not able to run the configure cluster ID binary on GKE before the allow list change?

Reply:
We needed to update this so that the script is an exact match with the AllowlistWorkload. I double checked with Alphonse and cluster id is available in the current version of the sensor image